src/aap_eda/core/models/user.py

```diff
@@ -32,6 +34,17 @@ class User(AbstractUser):
     roles = models.ManyToManyField("Role", related_name="users")
     modified_at = models.DateTimeField(auto_now=True, null=False)
     is_service_account = models.BooleanField(default=False)
+    organizations = models.ManyToManyField(
```

Comment:
I don't want to add organizations to the User model, because that is tracked with an organization role. That role doesn't exist yet, the idea is that a migration will create it. DAB RBAC has some utilities for this but I don't plan to use it for EDA because it doesn't use the Django Meta to specify permissions. So plan is for those roles to be the same as current roles, python / dict-defined

Reply:
I think moving it to the organization model would essentially be the same thing and I guess we wouldn't want that:
```
class Organization(AbstractOrganization):
    users = models.ManyToManyField(
        settings.AUTH_USER_MODEL,
        related_name="organizations",
        help_text="The list of users in this organization.",
    )

```

Reply:
I'm saying don't move it to the organization model. Just delete this field. In the data migration for creating roles (`RoleDefinition` in dab_rbac) we will create a role for what Gateway sees as the `Organization.users` relationship.

Then in the EDA auth class for JWT consumption, the org-user mapping will be applied to the applicable RoleDefinition. This already exists in a partial form, with the existing EDA `Role` model here.

https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/jwt_consumer/eda/auth.py